docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+services:
+  postgres:
+    image: "postgres:14.1"
+    volumes:
+      - /var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: odkbuild
```

Comment:
Credentials hard coded here and in config.yml - could be parameterised via ENV. Seems good enough for Central though.